.env

```diff
@@ -1,9 +1,11 @@
 ## Set pro options if applicable
-RUNDECK_IMAGE=rundeckpro/enterprise:4.10.0
+RUNDECK_IMAGE=rundeckpro/enterprise:4.11.0
 RUNDECK_LICENSE_FILE=./rundeck-enterprise.key
 RUNDECK_USER=admin
 RUNDECK_PASSWORD=admin
 
+BUILD_DEV_IMAGE=rundeckpro/env-builder:latest
```

Comment:
We should version the env-builder image. That way if we make changes and push a new "latest" that also require changes here this won't break.